sandbox/grommet-app/src/pages/layouts/kinds/OpsRampDetail/QuickFilters.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useContext } from 'react';
+import { SelectorGroup, Selector } from '../../../../components';
+import {
+  StatusCriticalSmall,
+  StatusGoodSmall,
+  StatusUnknownSmall,
+  StatusPlaceholderSmall,
+} from 'grommet-icons';
+import { Box, DataContext, Text } from 'grommet';
+
+const MiniSelector = ({ value, count, icon, title }) => {
+  return (
+    <Selector
+      value={value}
+      title={
+        <Box align="center">
+          <Box direction="row" alignSelf="center" align="center" gap="xsmall">
```

Comment:
Is this alignSelf needed?

```suggestion
          <Box direction="row" align="center" gap="xsmall">
```

Reply:
no took out